src/api/api_auth_shibuqam.nit

```diff
@@ -94,7 +99,10 @@ class ShibCallback
 			config.players.save player
 		end
 		session.player = player
-		res.redirect "/player"
+
+		# Redirect
+		var page = session.auth_next or else "/player"
```

Comment:
My bad, I though it was this one. It appears that ShibUqam and Github does not have a common ancestor more specific than APIHandler...

But I don't like the id of putting everything in APIHandler, it makes it API dirtier :/